judgels-frontends/raphael/src/routes/problems/problemsets/single/problems/single/submissions/ProblemSubmissionRoutes.jsx

```diff
@@ -17,7 +17,7 @@ function ProblemSubmissionRoutes() {
 
         <Route
           path="/problems/:problemSetSlug/:problemAlias/submissions/:submissionId"
-          component={ProblemSubmissionPage}
+          render={props => <ProblemSubmissionPage {...props} problemJid={problemJid} />}
```

Comment:
Actually, no need to do all this. `problemJid` is already available in Redux store. We can cell `selectProblemSetProblem()` in `mapStateToProps` in `ProblemSubmissionPage`. It is an object that contains `problemJid` 😄